### DISCUSSION TITLE
[PSD11] Exercise 3.3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#173] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210061872-89085825-1410-4de0-9e65-dace3715ab3e.png)

**Comment by Elia-Belli** [comment]
![211631080-ecf9c74a-977e-401d-8727-3f0aa29a339c](https://user-images.githubusercontent.com/107944476/213870160-8ae778fb-ed81-4a28-88ba-967d81b7c1d7.png)

Un D-Latch campiona il segnale D durante tutto l'intervallo in cui $CLK = 1$

**Comment by fraxb04** [comment]
@Elia-Belli 
corretto, ma in teoria Q nello stato iniziale Q (prima che si alzi il CLK) non dovrebbe essere indefinito, ovvero essere sia 1 sia 0.

**Comment by Elia-Belli** [comment]
Ho corretto, $Q$ prima del primo fronte positivo del clock è un valore sconosciuto compreso tra 0 e 1.
